matchzoo/contrib/layers/encoding_layer.py

```diff
@@ -0,0 +1,122 @@
+"""An implementation of EncodingModule for DIIN model."""
```

Comment:
I think `encoding layer` is required by almost every model, and the structures are not consistent. If it is difficult to write this layer in `DIIN` model, you'd better use another name.